docs/stable/1-client.md

```diff
@@ -344,14 +344,6 @@ There are several security considerations to take into account when running Stat
 
 ### Scalability and UX
 
-**Bandwidth usage:**
-
-In version 1 of Status, bandwidth usage is likely to be an issue. For more investigation into this, see [the theoretical scaling model](https://github.com/vacp2p/research/tree/dcc71f4779be832d3b5ece9c4e11f1f7ec24aac2/whisper_scalability).
```

Comment:
I'd keep this as it is still relevant in version 1. Replace second sentence with "In version <X> this is partially addressed with Waku usage

Reply:
FYI, I also added a reference to this in the new `9-waku-usage.md` document, under the `topic-interest` references.